### ISSUE TITLE
Run error

### ISSUE THREAD

**Issue by masonc15** [enzanki-ars/rocket-league-minimap-generator#35]
Hello,

I am getting the following error when trying to analyze an existing JSON.

```
C:\Users\darks\rocket-league-replay-analysis>python -m rocketleaguereplayanalysis.main test.json
Parsing Player Info: 100%|#####################################################| 7439/7439 [00:00<00:00, 309959.24it/s]
Parsing Frame Data: 100%|########################################################| 7439/7439 [00:01<00:00, 5862.10it/s]
Traceback (most recent call last):
  File "C:\Python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\darks\rocket-league-replay-analysis\rocketleaguereplayanalysis\main.py", line 129, in <module>
    main()
  File "C:\Users\darks\rocket-league-replay-analysis\rocketleaguereplayanalysis\main.py", line 88, in main
    load_data(args.game_json)
  File "rocketleaguereplayanalysis\data\data_loader.py", line 30, in load_data
    parse_pressure()
  File "rocketleaguereplayanalysis\util\extra_info.py", line 50, in parse_pressure
    field_dimensions = get_field_dimensions()
  File "rocketleaguereplayanalysis\util\extra_info.py", line 21, in get_field_dimensions
    x_size = ((x_w - (x_w % (2 * get_config('size_modifier')))) /
  File "rocketleaguereplayanalysis\util\config.py", line 27, in get_config
    return GenConst[key].value
TypeError: 'type' object has no attribute '__getitem__'
```

Any help would be appreciated.  I have installed all requirements and am running Windows 10.

**Comment by enzanki-ars** [comment]
Interesting issue.  I think I have an idea of where to look first.

**Comment by Slokh** [comment]
I think I was getting a similar issue. The problem I think is you installed the `enum` package, but you should install the `enum34` package. So just uninstall the first and the install the second.

**Comment by masonc15** [comment]
I think Slokh's response fixed that problem but I am having trouble installing cairosvg with pip.  Whenever I try to do it, I get the following error:

```
C:\Users\darks\rocket-league-replay-analysis>pip install cairosvg
Collecting cairosvg
  Using cached CairoSVG-2.1.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\darks\appdata\local\temp\pip-build-9x6b64\cairosvg\setup.py", line 24, in <module>
        with open(init_path, 'r', encoding='utf-8') as fd:
    TypeError: 'encoding' is an invalid keyword argument for this function

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in c:\users\darks\appdata\local\temp\pip-build-9x6b64\cairosvg\
```

**Comment by Slokh** [comment]
I also had a lot of trouble getting cairo to work. Install GTK for Windows using this installer, it will install cairo
https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases

**Comment by enzanki-ars** [comment]
In regards to the cairosvg problem, running a fresh windows install and python 3.6, I needed to also install something else, but I don’t remember of the top of my head. Sadly, I have not done extensive testing on other platforms like I should have, and I am not sure it works easily out side of my computer. I need to look into it more, so sorry about that. Got a little to trigger happy with showcasing the program before making is consumer friendly.

**Comment by masonc15** [comment]
Slokh, I tried installing GTK and restarting my computer but still no luck.  If I try to analyze a JSON, it says No module named cairosvg.  Any other tips?  Google does not have much help for this problem.

Enzanki, thanks a ton for your help so far.  This tool looks really cool so take your time in making it user-friendly.

**Comment by Slokh** [comment]
I think I installed an older version of cairosvg too `pip install cairosvg==1.0.22`

**Comment by enzanki-ars** [comment]
@masonc15 Are you able to run the `exe` under releases?  That should have `cairosvg` embedded in it.

**Comment by enzanki-ars** [comment]
I am looking into the issue on a fresh windows install.

**Comment by enzanki-ars** [comment]
Oh wow.  I really needed to test that.  Epic failure on my end.  Sorry folks.

**Comment by enzanki-ars** [comment]
I wonder if #36 will fix this...

**Comment by masonc15** [comment]
Hello,

I have two errors to report.  The first solution I tried was using the release as you said.  I received the following error:

```
C:\Users\darks\Downloads\rocketleaguereplayanalysis-windows>rocketleaguereplayanalysis --process_type video_all test.json
Traceback (most recent call last):
  File "rocket-league-replay-analysis-master\rocketleaguereplayanalysis\main.py", line 173, in <module>
  File "rocket-league-replay-analysis-master\rocketleaguereplayanalysis\main.py", line 88, in main
  File "rocket-league-replay-analysis-master\rocketleaguereplayanalysis\data\object_numbers.py", line 27, in parse_player_info
ModuleNotFoundError: No module named 'tqdm'
[14640] Failed to execute script main
```
I'm not sure why it can't recognize the tqdm module as I have installed this with pip already.

The second thing I tried was doing what Slokh said and installing an older version of Cairosvg.  This worked and got me past the main problem I had been dealing with but now I receive the following error:

```
C:\Users\darks\rocket-league-replay-analysis>python -m rocketleaguereplayanalysis.main --process_type video_all test.json
Parsing Player Info: 100%|#####################################################| 7439/7439 [00:00<00:00, 323434.76it/s]
Parsing Frame Data: 100%|########################################################| 7439/7439 [00:01<00:00, 5871.35it/s]
Minimap Render:   0%|                                                                         | 0/7439 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "C:\Python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\darks\rocket-league-replay-analysis\rocketleaguereplayanalysis\main.py", line 129, in <module>
    main()
  File "C:\Users\darks\rocket-league-replay-analysis\rocketleaguereplayanalysis\main.py", line 113, in main
    do_render_minimap(video_prefix)
  File "rocketleaguereplayanalysis\util\do_render.py", line 8, in do_render_minimap
    render_field(video_prefix)
  File "rocketleaguereplayanalysis\render\minimap.py", line 22, in render_field
    out_prefix=out_prefix)
  File "rocketleaguereplayanalysis\render\minimap.py", line 29, in render_frame
    import cairosvg
  File "C:\Python27\lib\site-packages\cairosvg\__init__.py", line 27, in <module>
    from . import surface
  File "C:\Python27\lib\site-packages\cairosvg\surface\__init__.py", line 30, in <module>
    import cairo  # pycairo
ImportError: No module named cairo
```

There isn't a standalone pip package called cairo so I'm not sure what the problem is.  Hopefully this information helps.

**Comment by enzanki-ars** [comment]
I am going to start work on removing all aspects of SVGs from the program, as it has been nothing but trouble for me. Gladly, FFmpeg seems to do what I need. 

As for the tqdm error, that is a result of some weird errors I noticed, I suspect from the failure to properly install all python modules. PyInstaller was reporting that it could not find any of the modules, causing them all to be missing from the final output.

**Comment by enzanki-ars** [comment]
In the meantime, it may still be possible to release a fixed version. I will check on that by compiling on my laptop where it seems to work.

**Comment by enzanki-ars** [comment]
@masonc15 @Slokh 

I have just released `v1.4.0-alpha1` at https://github.com/enzanki-ars/rocket-league-replay-analysis/releases/tag/v1.4.0-alpha1, though it is missing the minimap output.  Mind testing to see if that fixes the issues?
